lib/clearance.rb

```diff
@@ -10,7 +10,4 @@
 require 'clearance/constraints'
 
 module Clearance
```

Comment:
Can we remove this `module` too, since everything is already loaded by this point?